src/routes/choose-outfit/+page.svelte

```diff
@@ -4,16 +4,26 @@
 	import Popover from '$lib/components/Popover/Popover.svelte';
 	import PopoverMultiSelectContent from '$lib/components/Popover/CustomPopoverContent/PopoverMultiSelectContent.svelte';
 	import PopoverChipTrigger from '$lib/components/Popover/CustomPopoverTrigger/PopoverChipTrigger.svelte';
-	import { outfitStore, clothesStore, weatherStore } from '$lib/utilities/stores';
+	import StandardContentLayout from '$lib/components/StandardContentLayout/StandardContentLayout.svelte';
+	import {
+		outfitStore,
+		clothesStore,
+		weatherStore,
+		selectedOutfitId
+	} from '../../lib/utilities/stores';
+	import Tabs from '$lib/components/Tabs/Tabs.svelte';
+	import TabContent from '$lib/components/Tabs/TabContent.svelte';
+	import TabLabel from '$lib/components/Tabs/TabLabel.svelte';
+	import Toast from '$lib/components/Toast/Toast.svelte';
 
 	let popoverItems = [];
-	let isOutfitShown = false;
-	let filteredOutfits = [];
-	let isOutfitSubmitted = false;
+	let filteredOutfits = $outfitStore;
+	let toast;
+	let tabs;
+	let activeTabNumber = $selectedOutfitId === undefined ? '1' : '2';
 
 	// The DEFAULT temperature is 50 degrees unless specified to use current temp
 	let clothingFilters = { cozy: false, formal: false, temp: 50 };
```

Comment:
If the "cozy" and "formal" filters are not specified, does that only show outfits that do NOT have those tags set, or do the filters only exclude outfits? The latter solution seems to make more sense to me, but it seems like it is the former, since selecting the "formal" filter causes two outfits to be shown, while by default only one is shown

Reply:
By default, it shows outfits that do not have the cozy or formal tag set (so when cozy and formal == false), which by default only shows one outfit.

Reply:
resolved